dist/images/Dockerfile.fedora

```diff
@@ -15,7 +15,7 @@ USER root
 
 ENV PYTHONDONTWRITEBYTECODE yes
 
-ARG ovnver=ovn-20.12.0-25.fc33
+ARG ovnver=ovn-21.03.0-2.fc33
```

Comment:
this commit should be ordered before the commit that enables the new behavior

Reply:
Reordered